ee/tabby-db/src/user_completions.rs

```diff
@@ -61,6 +68,71 @@ impl DbConn {
         Ok(())
     }
 
+    // FIXME(boxbeam): index `created_at` in user_completions table.
+    pub async fn compute_daily_stats_in_past_year(
+        &self,
+        users: Vec<i32>,
+    ) -> Result<Vec<UserCompletionDailyStatsDAO>> {
+        let users = users
+            .iter()
+            .map(|u| u.to_string())
+            .collect::<Vec<_>>()
+            .join(",");
+        Ok(sqlx::query_as(
+            r#"
+        SELECT CAST(STRFTIME('%s', DATE(created_at)) AS TIMESTAMP) as start,
+               SUM(1) as completions,
+               SUM(selects) as selects
+        FROM user_completions
+        WHERE created_at >= DATE('now', '-1 year')
+            AND (? = '' OR user_id IN (?))
+        GROUP BY 1
+        ORDER BY 1 ASC
+        "#,
+        )
+        .bind(&users)
+        .bind(&users)
```

Comment:
This seems like a mistake, did you mean to bind the same thing twice? If so, use `$1` in the query and bind once